spec/examples/pay_with_provider.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class PayWithProvider < Actor
-  input :provider, in: %w[MANGOPAY PayPal Stripe], default: "Stripe"
+  input :provider, inclusion: %w[MANGOPAY PayPal Stripe], default: "Stripe"
```

Comment:
This can be reverted back to the previous definition, since in the main branch I allow both syntaxes. The new syntax is tested with another test example:

https://github.com/sunny/actor/blob/f47462b/spec/examples/pay_with_provider_inclusion.rb

```suggestion
  input :provider, in: %w[MANGOPAY PayPal Stripe], default: "Stripe"
```

Reply:
The previous `in` is now deprecated. It is supported, but is it better to demonstrate the new syntax in the documentation?

Reply:
Yeah, the documentation in the README should definitely promote `inclusion` in favor of `in`. However, as long as the `in` is still supported it’s good to have at least one test that uses the previous syntax.

Reply:
Yeah, the documentation in the README should definitely promote `inclusion` in favor of `in`. However, as long as the `in` is still supported it’s good to have at least one test that uses the previous syntax.

Reply:
Understood. I agree. I saw that there is `PayWithProviderInclusion` for the new keyword. Applied.